code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -953,7 +953,6 @@
 	connected_ai = mainframe
 	mainframe.connected_robots |= src
 	lawupdate = TRUE
-	lawsync()
```

Comment:
This is being called twice. I have no idea where else it's being called so I'm removing this one.